Resources/config/android.xml

```diff
@@ -25,6 +25,7 @@
             <argument>%rms_push_notifications.android.gcm.use_multi_curl%</argument>
             <argument>%rms_push_notifications.android.timeout%</argument>
             <argument type="service" id="logger" />
+            <argument>%rms_push_notifications.android.dry_run%</argument>
```

Comment:
I had to add:
`<argument>null</argument>`

before the `dry_run` argument because otherwise I get a 

`PHP Fatal error:  Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Type error: Argument 5 passed to RMS\PushNotificationsBundle\Service\OS\AndroidGCMNotification::__construct() must be an instance of Buzz\Client\AbstractCurl, boolean given`